### DISCUSSION TITLE
Custom headers

### DISCUSSION THREAD

**Discussion by PatelN123** [vercel/community#381] [Help] [answered]
Hey,

This has been troubling me for a few days. 

In netlify I would just create a file called `_headers` and put the following code inside. 

```
/sitelogo.svg
  content-disposition: attachment; filename="sitelogo.svg"
 ```
 
I'm wondering what the alternative for vercel is? 

I've messaged Vercel support directly, they don't seem to know and I have tried searching around online. - Is this not a feature in vercel or am I missing something obvious?

**Comment by leerob** [comment, marked as answer]
**vercel.json**

```json
{
  "headers": [
    {
      "source": "/sitelogo.svg",
      "headers" : [
        {
          "key" : "content-disposition",
          "value" : "attachement; filename=\"sitelogo.svg\""
        }
      ]
    }
  ]
}
```

This is [listed in our docs](https://vercel.com/docs/cli#project-configuration/headers) - anything we could do better to surface this?

**Reply by PatelN123** [reply]
Ah yes thank you. 
Yeah, I didn't quite understand what was supposed to go in the key and value. 

Thanks
